### PR TITLE
HW1 (hj2589)

### PR DESCRIPTION
**Pull request by katehyerinjeon**

### CODE REVIEW

File: contributions/pydigree.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Pydigree
```
Comment (timkpaine) [inline contributions/pydigree.md:1]: move to `projects/python/pydigree.md`

File: project/pynrm.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# `pynrm`
```
Comment (timkpaine) [inline project/pynrm.md:1]: move to `projects/python/pynrm.md`

File: tkp2108.md [hunk 3]
```diff
@@ -1,14 +0,0 @@
-# Open Source Development
```
Comment (timkpaine) [inline tkp2108.md:1]: restore `tkp2108.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by katehyerinjeon** [comment]
Closed and reopened a new pull request using hw1 branch

**Pull request closed without merging**